lib/zht/LocationProvider.dart

```diff
@@ -1,9 +1,11 @@
 import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
 import 'package:location/location.dart';
 
 class LocationProvider with ChangeNotifier {
   LocationData _currentLocation;
   LocationData _startingPosition;
+  String  _value;
```

Comment:
Why store the boundary setting in the LocationProvider ?
Please create a SettingsProvider